src/mikro-orm-pgis/geometry/MultiPolygon.ts

```diff
@@ -0,0 +1,16 @@
+import { Type } from "@mikro-orm/core";
+import { MultiPolygon } from "geojson";
+import { SimpleFeature } from "..";
+
+export class MultiPolygonGeomType extends Type<MultiPolygon, string> {
```

Comment:
Are you planning to add methods like `convertToJSValue`, `convertToJSValueSQL`, etc in a future PR? I think we need them here but I can see why you would decide to hold off on implementing them until we actually have data to play with.

Reply:
Yes and agreed. I made a [note on the original ticket](https://github.com/NYCPlanning/ae-zoning-api/issues/10#issuecomment-1787896263) but should have linked it here.